AzureCalling/app/src/main/java/com/azure/samples/communication/calling/view/ParticipantView.java

```diff
@@ -59,7 +60,11 @@ public void setVideoStream(final RemoteVideoStream remoteVideoStream) {
 
         try {
             final VideoStreamRenderer videoRenderer = new VideoStreamRenderer(remoteVideoStream, getContext());
-            setVideoRenderer(videoRenderer);
+            if (MediaStreamType.SCREEN_SHARING == remoteVideoStream.getMediaStreamType()) {
```

Comment:
Should probably swap the equality expressions, since all of our other conditions go that way

Reply:
addressed.